css/blogpost.scss

```diff
@@ -0,0 +1,84 @@
+@import 'styleguide';
+
+div {
+  font-family: $font-body;
+}
+
+p {
+  line-height: 2em;
+  font-size: 1em;
+  color: $offblack;
+}
+
+.title {
+  font-family: $font-heading;
+  font-weight: 800;
+  font-size: 3em;
+  color: $offblack
+}
+
+.subheading {
+  font-family: $font-heading;
+  font-weight: 600;
+  font-size: 2em;
+  margin-top: 1em;
+  margin-bottom: -.5em;
+  color: $darkgrey;
+}
+
+.date {
+  color: $grey;
+  font-size: 1.5em;
+  margin: .5em 0em;
+  font-family: $font-accent;
+}
+
+.post {
+  margin: 1em auto;
+  line-height: 1.5em;
+}
+
+.center {
+  margin: 0em auto;;
+}
+
+#container-top {
+  margin-top: 3em;
+}
+
+#container-bot {
+  margin-bottom: 6em;
+}
+
+.caption {
+  font-family: $font-accent;
+  text-align: center;
+  font-size: 1em;
+  color: #666;
```

Comment:
it's like the metal hand signal like
fuck yeah \m/
it's cuz the color u used was 666 hehe